recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -574,13 +663,23 @@ def main(argv):
         if args.force:
             # When forced, ignore all the checking and call the underlying updater directly
             updater = FirmwareUpdate(args.firmware)
+        if args.rollback:
+            updater = RollbackFirmware(args.firmware)
```

Comment:
How does rollback go along with the force mode?

Reply:
> How does rollback go along with the force mode?

they are mutex,  use two command parameters would show errors at the same time thrown by `argparse`

Reply:
OK, that's fine.